File: OOPDraw/EquliateralTriangle.cs
using Nakov.TurtleGraphics;
using System;
using System.Drawing;

namespace OOPDraw
{
    public class EquliateralTriangle : Shape
    {
        //Properties
        public float Side { get; set; }

        //The 'Constructor
        public EquliateralTriangle(float originX, float originY, float side) : base(originX, originY)
        {
            Side = side;
        }

        public override void Draw()
        {
            ResetTurtle();
            Turtle.Rotate(30);
            for (int i = 0; i < 3; i++)
            {
                Turtle.Forward(Side);
                Turtle.Rotate(120);
            }
        }

        public override void Resize(float x, float y)
        {
            //Ignore Y
            Side = x;
        }
    }
}

File: OOPDraw VB/OOPDraw/EquliateralTriangle.vb
Imports Nakov.TurtleGraphics

Public Class EquliateralTriangle
    Inherits Shape
    'Properties
    Private Property SideLength As Single

    'The 'Constructor'
    Public Sub New(xOrigin As Single, yOrigin As Single, sideLength As Single)
        MyBase.New(xOrigin, yOrigin)
        Me.SideLength = sideLength
    End Sub

    Public Overrides Sub Draw()
        ResetTurtle()
        Turtle.Rotate(30)
        For i = 1 To 3
            Turtle.Forward(SideLength)
            Turtle.Rotate(120)
        Next
    End Sub

    Public Overrides Sub Resize(x As Single, y As Single)
        'Ignore y
        SideLength = x
    End Sub
End Class
